src/components/seekbar-preview/seekbar-preview.js

```diff
@@ -18,41 +21,47 @@ const mapStateToProps = state => ({
  * @class SeekBarPreview
  * @extends {Component}
  */
+@withPlayer
 @connect(mapStateToProps)
 class SeekBarPreview extends Component {
   /**
-   * utility function to get the thumb sprite background position
-   * @returns {string} background-position string value
+   * Gets the style of the frame preview image.
+   * @param {Object} thumbnailInfo  - The thumbnail info data.
+   * @returns {string} - The css style string.
+   * @private
    */
-  getThumbSpriteOffset(): string {
-    const percent = this.props.virtualTime / this.props.duration;
-    const sliceIndex = Math.ceil(this.props.thumbsSlices * percent);
-    return -(sliceIndex * this.props.thumbsWidth) + 'px 0px';
+  _getFramePreviewImgStyle(thumbnailInfo: Object): string {
+    let framePreviewImgStyle = `background: url(${thumbnailInfo.url});`;
```

Comment:
it was `background-image: url....` what's the difference?

Reply:
Just different syntax since we don't need previous styles anymore.
In a background property you can add `background-color`, `repeat`, `no-repeat` and other image attributes, but in the background-image property you are only allowed to add image.

```
background-image: url("");
background-repeat: no-repeat;
background-position: right top;
```

and in background property you can do in one line all these

`background: #ccc url("") no-repeat;`